nodedoc/nodeset/src/main/java/at/scch/nodedoc/nodeset/StructuredBrowseName.java

```diff
@@ -0,0 +1,32 @@
+package at.scch.nodedoc.nodeset;
+
+import lombok.Getter;
```

Comment:
Die Verwendung von Lombok impliziert, dass ein Developer auch den entsprechenden Support in der IDE haben muss?
Wenn ich das richtig verstehe, spart das Boilerplate Code?